packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -298,8 +298,8 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 		return {
 			selectedPlan: plansStore.getPlanByProductId( selectedPlanProductId, locale ),
 			selectedPaidPlan: plansStore.getPlanByProductId( selectedPaidPlanProductId, locale ),
-			selectedPlanBillingPeriod: plansStore.getPlanProductById( selectedPlanProductId )
-				?.billingPeriod,
+			selectedPlanBillingPeriod:
```

Comment:
Ignore what I previously wrote about naming this `selectedPlanBillingPeriod` in https://github.com/Automattic/wp-calypso/pull/49412#discussion_r567601827

`billingPeriod` would be a better name since it can be derived from selected plan if there is a selected plan. Otherwise it's `'ANNUAL'`.